src/test/java/com/mathworks/ci/RunMatlabBuildBuilderTest.java

```diff
@@ -129,6 +129,7 @@ public void verifyMATLABlaunchedWithDefaultArguments() throws Exception {
         this.buildWrapper.setMatlabBuildWrapperContent(new MatlabBuildWrapperContent(Message.getValue("matlab.custom.location"), getMatlabroot("R2017a")));
         project.getBuildWrappersList().add(this.buildWrapper);
         scriptBuilder.setTasks("");
+        scriptBuilder.setBuildOptions(new BuildOptions(""));
```

Comment:
I think we could structure this so that the build options don't need to be set when they're empty. For reference of how this would look here's how that's handled for startup options:

```
String options = getStartupOptions() == null ? "" : getStartupOptions().getOptions();
```

Also, make sure that we have at least one test where the build options aren't set to correlate with the case where the user doesn't even check the build options box. What is the behavior at the moment if you don't specify build options via the UI? I don't see any null checks so it seems like there's probably a null pointer error?

Reply:
Yes, updated the code with null checks. Also, updated the tests with unspecified build options.